config/default/common/config/metadata/layers/viirs/noaa21/VIIRS_NOAA21_Brightness_Temp_BandI5_Day.md

```diff
@@ -0,0 +1,7 @@
+The VIIRS Brightness Temperature (Band I5, Day) layer is the brightness temperature, measured in Kelvin (K), calculated from the top-of-the-atmosphere radiances. It does not provide an accurate temperature of either clouds nor the land surface, but it does show relative temperature differences which can be used to distinguish features both in clouds and over clear land. It can be used to distinguish land, sea ice, and open water over the polar regions during winter (in cloudless areas).
```

Comment:
"...either clouds nor..." should be "...either clouds or..." or "...neither clouds nor..."